client/src/pages/UserLanding/UserLanding.jsx

```diff
@@ -46,15 +51,29 @@ const UserLanding = () => {
 
   const handleSubmit = async (e) => {
     e.preventDefault();
-    // const includesStrings = /workout|women/.test(userSearchTerm);
-    // const searchQuery = userSearchTerm + "workout" + "women";
-    // const searchResults = await getSearchVideos(searchQuery);
+
+    // const searchResults = await getSearchVideos(userSearchTerm);
```

Comment:
Oh interesting! Is your initial search not actually hitting the youtube API? Why continue to use the mock data below?